### DISCUSSION TITLE
Can i use the same email that was linked to another account earlier in creating a new github account ?

### DISCUSSION THREAD

**Discussion by yocki8** [community/community#131646] [Other Feature Feedback, Questions, & Ideas]
See I created a github account using my organization given email.. But Github flagged my account for no reason and now my account is hidden and not accessible to anyone publicly. I also raised a reinstatement request but some guys are saying that it'll take a long time for github support to reply. And also I can't delete my flagged account. So I am thinking to gave this flagged account a new dummy email id and unlink my primary organization email to use it to create a new account. Is this idea executable or not ??

**Comment by gaomingzhao666** [comment]
**Sadly say that, it seems can not create a another `GitHub account` by using one email address.**[**here is a case**](https://stackoverflow.com/questions/38445289/how-to-create-a-new-github-account-with-the-same-email-address#:~:text=You%20can't%20create%20more,with%20a%20new%20email%20id%20.)

**But you can try to submit a report for `GitHub team`, and they probably can solve this problem.**

If my comment is help you solve this question, please mark my comment as answer, thanks bro. 😊

**Reply by yocki8** [reply]
<img width="617" alt="image" src="https://github.com/community/community/assets/89199719/d95d62b5-09b0-4188-8020-32aad278c785">

I am not saying to use it for both accounts at same time.. I was asking to use it **AFTER UNLINKING IT WITH FLAGGED ACCOUNT**.

**Reply by gaomingzhao666** [reply]
**I am so sorry about that my misunderstood it, I think if you can access that account now, this solution seems can work, because you must login and validate that email address.**

**Reply by yocki8** [reply]
But I have a fear that what if I unlink it from here.. and then new account creation will say "this email is already linked ( or it can't be used again )". If this happens I will lose hope from all sides and might lose my new Internship 🥲

**Reply by gaomingzhao666** [reply]
I can't say that certainly, so if this account is so important to you, I think you can try to take a report to `GitHub team` even it is going to be waste a long time and maybe still can not solve this problem.

[Here is the statement post by `GitHub team`](https://github.com/github/docs/issues/33722)
> We may be slow to review contributions from June 28th through July 8th, as many people on the docs team will be taking time off.

**Reply by TheGiraffe3** [reply]
It is July 10th.
It is executable if you want, but I don't think that the internship will be transferred. Neither will your roles in the organization.

**Reply by yocki8** [reply]
> It is July 10th. It is executable if you want, but I don't think that the internship will be transferred. Neither will your roles in the organization.

and also tell if I am able to use this flagged account  SSH keys in new account ??

**Reply by TheGiraffe3** [reply]
No, you'd have to make new ones.
